src/math/spacegroup.cpp

```diff
@@ -230,11 +234,18 @@ namespace OpenBabel
                             t = &v.z();
                             break;
                           }
+                        // Cover cases like 7/6
+                        double numerator = (double) (row[j] - '0');
+                        double denumerator = (double) (row[j+2] - '0');
+                        if (numerator > denumerator)
+                          numerator -= ((int)numerator % (int)denumerator) * denumerator;
+
                         *t = ((double) (row[j] - '0')) / (row[j+2] - '0');
                         if (neg)
                           *t = - *t;
+
+                        j +=2;
                       }
-                    j +=2;
```

Comment:
I think I'd rather have a better detection for this case, rather than just incrementing. Can we instead check if the next character is '-' rather than removing this for all cases?